### PR TITLE
Test: Install language packs programmatically in Behat tests, solves …

### PR DESCRIPTION
**Pull request by lucaboesch**
…#540.

### CODE REVIEW

**Comment by lucaboesch** [comment]
It did pass like this. Thanks very much @abias you seem to have done the magic.

File: tests/behat/theme_boost_union_smartmenusettings_menuitems_presentation.feature [hunk 1]
```diff
@@ -17,6 +17,9 @@ Feature: Configuring the theme_boost_union plugin on the "Smart menus" page, app
     And I create smart menu with the following fields to these values:
       | Title            | Quick links              |
       | Menu location(s) | Main, Menu, User, Bottom |
+    And the following "language packs" exist:
+      | language |
+      | de       |
```
[the remark this answers is not in the document]
Comment (lucaboesch) [reply tests/behat/theme_boost_union_smartmenusettings_menuitems_presentation.feature]: Agreed.

File: tests/behat/theme_boost_union_smartmenusettings_menus_presentation.feature [hunk 2]
```diff
@@ -8,6 +8,9 @@ Feature: Configuring the theme_boost_union plugin on the "Smart menus" page, app
     Given the following "users" exist:
       | username |
       | user1    |
+    And the following "language packs" exist:
+      | language |
+      | de       |
```
[the remark this answers is not in the document]
Comment (lucaboesch) [reply tests/behat/theme_boost_union_smartmenusettings_menus_presentation.feature]: Agreed.

**Comment by abias** [comment]
I am glad that I could help :)
Merging.

**Review by abias — APPROVED** [review]

**Pull request merged**